doc/manual/manual.tex

```diff
@@ -2045,18 +2053,26 @@ \subsection{Installing and running \aspect{} on Mac OS X}
 \end{verbatim}}
 \dealii{} app opens a terminal window and displays a \dealii{} message.   \dealii{}.app will install all required libraries for \aspect{} (p4est, parMeTiS, and Trilinos) and will include the environment variables needed to use these libraries.
 
+To ensure the correct compilers are picked up when configuring \aspect{}, add the following lines to your .profile or .bash\_profile:
+
+\begin{verbatim}
+export OMPI_CXX=clang++
+export OMPI_CC=clang
```

Comment:
I'm not sure of the exact rules here - some seem to be indented by 2 spaces, some by 1, and some by none?